src/calculator/main.cpp

```diff
@@ -0,0 +1,161 @@
+#include "../build/rmkit.h"
+#include <string>
+#include <vector>
+#include <algorithm>
+#include <stdlib.h>
+#include "calc.h"
+
+class CalcDisplay: public ui::Text {
+  public:
+   CalcDisplay(int width, int height)
+    : Text(0, 0, width, height, "") {}
+};
+
+StackElement::StackElement(int width, int height) {
+    content = "";
+    display = new CalcDisplay(width, height);
+    ui::Style format = ui::Stylesheet()
+        .font_size(64)
+        .line_height(1.2)
+        .underline(false)
+        .justify_right()
+        .valign_middle()
+        .border_all()
+        .build();
+    display->set_style(format);
```

Comment:
re: styles -- set_style actually takes ui::Stylesheet as its argument, not ui::Style, though there's a conversion defined between Style and Stylesheet. The idea is that Stylsheet can be used either as a builder for Style (like this), or for partial updates to an existing style. What you have works and is totally fine, but FYI it could be simplified to

```cpp
// line_height(1.2).underline(false) are defaults, so they aren't necessary to set again
display->set_style(ui::Stylesheet()
    .font_size(64)
    .justify_right()
    .valign_middle()
    .border_all());
```